pkg/setting/setting.go

```diff
@@ -79,18 +79,23 @@ var (
 	LogConfigs []util.DynMap
 
 	// Http server options
-	Protocol           Scheme
-	Domain             string
-	HttpAddr, HttpPort string
-	SshPort            int
-	CertFile, KeyFile  string
-	SocketPath         string
-	RouterLogging      bool
-	DataProxyLogging   bool
-	DataProxyTimeout   int
-	StaticRootPath     string
-	EnableGzip         bool
-	EnforceDomain      bool
+	Protocol                       Scheme
+	Domain                         string
+	HttpAddr, HttpPort             string
+	SshPort                        int
+	CertFile, KeyFile              string
+	SocketPath                     string
+	RouterLogging                  bool
+	DataProxyLogging               bool
+	DataProxyTimeout               int
+	DataProxyTlsHandshakeTimeout   int
```

Comment:
Acronyms should have [consistent case](https://github.com/golang/go/wiki/CodeReviewComments#initialisms) in Go:
```suggestion
	DataProxyTLSHandshakeTimeout   int
```